services/management/dbaas/psmdb_cluster_service.go

```diff
@@ -102,21 +101,38 @@ func (s PSMDBClusterService) ListPSMDBClusters(ctx context.Context, req *dbaasv1
 
 // GetPSMDBCluster returns a PSMDB cluster by name.
 func (s PSMDBClusterService) GetPSMDBCluster(ctx context.Context, req *dbaasv1beta1.GetPSMDBClusterRequest) (*dbaasv1beta1.GetPSMDBClusterResponse, error) {
-	_, err := models.FindKubernetesClusterByName(s.db.Querier, req.KubernetesClusterName)
+	kubernetesCluster, err := models.FindKubernetesClusterByName(s.db.Querier, req.KubernetesClusterName)
 	if err != nil {
 		return nil, err
 	}
 
 	// TODO: implement on dbaas-controller side:
-	// 1. Get psmdb host and status
+	// 1. Done in PMM-6899 --- Get psmdb host and status ---
```

Comment:
let's remove done part from the PR

Reply:
by done part I meant what have done by you. 
TODO should be up to date

Reply:
Done